main.cpp

```diff
@@ -92,6 +92,129 @@ good to go!
  */
 
 #include <iostream>
+
+ /*4) make them return the correct primitive type. e.g. if you're implementing the FloatType::add function, your implementation would start like this:
+        float FloatType::add( float lhs, float rhs ) */
+
+#include <math.h>
+#ifdef INFINITY
+#endif
+
+
+///////// FLOAT  ...............
+///////// 
+class FloatType 
+{
+    public:
+        float lhs, rhs;
+        float add(float, float);
+        float subtract(float, float);
+        float multiply(float, float);
+        float divide(float, float);
+};
```

Comment:
use `struct` not `class`
Accessor keywords have not been explained at this point in the course.
`struct` is `public` by default
Fix the indentation when you remove the word `public` too.
- [ ] revise